packages/react-native-snap-carousel/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-native-snap-carousel",
+  "version": "0.1.0",
+  "scripts": {
+    "build": "bsb -make-world",
+    "start": "bsb -make-world -w",
+    "clean": "bsb -clean-world"
+  },
+  "keywords": [
+    "BuckleScript"
+  ],
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "bs-platform": "^5.0.6"
+  },
+  "dependencies": {
+    "reason-react": "^0.7.0",
+    "reason-react-native": "^0.60.0"
```

Comment:
these need to be peer-dependencies and devDependencies at the same time
https://dev.to/btdev/what-are-those-peerdepenedencies-in-a-nodejs-project-51jo

Reply:
and react-native-snap-carousel as peer-dependency only